Dockerfile

```diff
@@ -1,6 +1,4 @@
-FROM php:7.0-fpm-stretch
-
-RUN echo "deb http://archive.debian.org/debian stretch main" > /etc/apt/sources.list
```

Comment:
instead of waiting for this to happen maybe we should include the archive for bullseye to future proof the container building

Reply:
Unfortunately until a dist is placed into archive it is not in archive so adding this for bullseye is causing an error in the docker build process